### PR TITLE
Feedback link

### PR DESCRIPTION
**Pull request by kulmann**
## Description
This PR adds a feedback link to the topbar which links to a survey.

## Motivation and Context
Gather user feedback.

## Screenshots (if appropriate):
Screenshot 2021-07-07 at 13 48 03

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: packages/web-runtime/src/components/FeedbackLink.vue [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <oc-icon name="feedback" />
+  </oc-button>
+</template>
+
+<script>
+export default {
+  name: 'FeedbackLink',
+  computed: {
+    href() {
+      return 'https://owncloud.com/web-design-feedback'
```
Comment (pascalwengerter) [inline packages/web-runtime/src/components/FeedbackLink.vue:19]: Not sure if out of scope but we could load it from the theme so users can customize and re-use it?

**Review by pascalwengerter — CHANGES_REQUESTED** [review]
Looks good code-wise but needs an updated snapshot and perhaps updated VRT files, too!

File: packages/web-runtime/src/components/FeedbackLink.vue (hunk 1 above)
Comment (kulmann) [reply packages/web-runtime/src/components/FeedbackLink.vue:19]: If someone wants to have this as a feature we could do so. For now we are the ones who want that user feedback. ;-) So I'd start with the hardcoded link and see how it evolves.

**Comment by ownclouders** [comment]
Results for <strong>oCISBasic</strong> https://drone.owncloud.com/owncloud/web/17149/45/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17151/49/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by fschade** [comment]
This is great, in my opinion this is good for now. In the future we could handle those things by using portals. LGMT 🚀

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17154/49/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Review by pascalwengerter — APPROVED** [review]
LGTM

**Pull request merged**